### PR TITLE
Fix Horse Mode crashing the game

### PR DESCRIPTION
**Pull request by PEPPERcula**

### CODE REVIEW

File: Patches/ChatCommandPatch.cs [hunk 1]
```diff
@@ -1,3411 +1,3411 @@
...
-        Directory.CreateDirectory(vipTagsFiles);
-        Directory.CreateDirectory(sponsorTagsFiles);
-
-        if (Blackmailer.CheckBlackmaile(PlayerControl.LocalPlayer) && PlayerControl.LocalPlayer.IsAlive())
-        {
-            goto Canceled;
-        }
-        switch (args[0])
-        {
+using Assets.CoreScripts;
```
Comment (NikoCat233) [inline Patches/ChatCommandPatch.cs:1]: Fix this

**Comment by NikoCat233** [comment]
image

**Comment by NikoCat233** [comment]
I dont know why you cant fix your chatcommandpatch
So i have to manually redo it

**Comment by NikoCat233** [comment]
Do not include mulitple features in a single pr next time

**Comment by NikoCat233** [comment]
Seems like you denied maintainers from editing your pr
Enable it pls

**Comment by NikoCat233** [comment]
bro your fork cant litterly build

**Comment by NikoCat233** [comment]
image

**Comment by PEPPERcula** [comment]
Idk why this happened but from before i only edited these lines and thats it. I didn't expect the whole file to change.
Screenshot 2024-12-01 081202

**Review by NikoCat233 — CHANGES_REQUESTED** [review]
Remove your changes in ChatCommandPatch
It cant be reviewed and it cant build

**Pull request merged**